tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_rocm5.3.2

```diff
@@ -0,0 +1,295 @@
+ARG BASEIMAGE=centos:7
```

Comment:
Is it possible to avoid copy/paste the code? Do you really need to create a new docker file for each ROCM version?

Reply:
It's not same as before. We use rocm/dev-centos-7:x.x.x as base image before. But we always waiting for it's release. We try to build this part as a stage by ourselves. line11~line102 is totally same as https://github.com/RadeonOpenCompute/ROCm-docker/blob/master/dev/Dockerfile-centos-7 which provided by AMD. Do you suggest to simplify it to suit our needs or keep it unchange?

Reply:
You build the code for ROCM 5.2.3, 5.3 and 5.3.2. Do you need three docker files, or just one?

Reply:
we can combine rocm5.2.3 and rocm5.3 as one because they similar, but for new version rocm5.3.2, it different so we need to keep it another file.

Reply:
Fine. Hopefully in the future we can merge them all into one.